libnavigation-core/api/current.txt

```diff
@@ -75,6 +77,11 @@ package com.mapbox.navigation.core {
     field public static final com.mapbox.navigation.core.MapboxNavigationProvider INSTANCE;
   }
 
+  public interface NavigationVersionSwitchObserver {
```

Comment:
Do we really want to expose this publicly? 🤔

Reply:
customer might need to change UI when sdk in a fallback mode